docs/reference/server-configuration.md

```diff
@@ -665,7 +775,7 @@ associated with starting each new query. For more information, see
 
 ## `ksql.query.pull.enable.standby.reads`
 
-**Per query:** yes
+**Per query:** yes (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

Comment:
@rodesai given how many of these configs which were labeled as per-query are absolutely not query-level I'm now skeptical that any of these actually do work for individual queries, but this one at least in theory _could_ be applied at the query level. Can you double check whether it actually works that way or not?

```suggestion
**Per query:** no (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

If it's _not_ a query-level config, then it at least would be a system property (mutable with `ALTER SYSTEM`) and not a server property, right?

Reply:
@JimGalasyn  I think this one is also supposed to be a feature flag, in which case it should also be a server property, ie just
```
**Per query:** no 
```